sdk/Sdk.Generators/Constants.cs

```diff
@@ -34,6 +34,9 @@ internal static class Types
             internal const string HttpResponse = "Microsoft.Azure.Functions.Worker.Http.HttpResponseData";
             internal const string HttpTriggerBinding = "Microsoft.Azure.Functions.Worker.HttpTriggerAttribute";
 
+            internal const string InputConverterAttributeType = "Microsoft.Azure.Functions.Worker.Converters.InputConverterAttribute";
+            internal const string SupportedConverterTypeAttributeType = "Microsoft.Azure.Functions.Worker.Converters.SupportedConverterTypeAttribute";
+            internal const string SupportsDeferredBindingAttributeType = "Microsoft.Azure.Functions.Worker.Extensions.Abstractions.SupportsDeferredBindingAttribute";
```

Comment:
You can also add these to [KnownFunctionMeatadataTypes](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/sdk/Sdk.Generators/FunctionMetadataProviderGenerator/KnownFunctionMetadataTypes.cs) and load/store the symbols for comparison later during parsing.